BUILD.gn

```diff
@@ -126,6 +126,20 @@ config("libdawn_public") {
   ]
 }
 
+# Executable needs an rpath to find our shared libraries on OSX
```

Comment:
nit: update comment with "OSX and Linux"

Reply:
done.